iOS/PluginClasses/PlayerViewController.swift

```diff
@@ -215,9 +215,11 @@ class PlayerViewController: UIViewController, IMAWebOpenerDelegate, PlaybackEven
             
             showPlaybackError()
         case kBCOVPlaybackSessionLifecycleEventPlaybackRecovered:
-            errorView?.removeFromSuperview()
-            isAdPlaybackBlocked = false
-            break
+            let reachabilityStatus = AFNetworkReachabilityManager.shared().networkReachabilityStatus
```

Comment:
For reachability events please ZAAppConnector..sharedInstance().connectivityDelegate that is aviable from ZappPlugins SDK.

Reply:
I have to update version of ApplicasterSDK to the latest (8.3.2) to use ZAAppConnector from ZappPlugins. But there is an error. 
![Screen Shot 2019-03-12 at 3 10 42 PM](https://user-images.githubusercontent.com/46022565/54199634-72238300-44da-11e9-8855-756b33a35522.png)
It reproduced even on empty projects. Can it be solved? Because in current version ZAAppConnector is not part of ZappPlugins and if I use it now I will have to rewrite it in future.